assets/model_monitoring/components/src/model_monitor_data_joiner/run.py

```diff
@@ -83,6 +83,12 @@ def run():
         args.right_join_column
     )
 
+    duplicate_column_names = joined_data_df.columns[joined_data_df.columns.duplicated()]
```

Comment:
this is just the names not the full column (including the content), right?

Reply:
correct